Examples/PSPDFCatalog/PSPDFCatalog/PSPDFCatalog.csproj

```diff
@@ -111,6 +111,15 @@
     <Reference Include="ZXingNetMobile">
       <HintPath>..\..\..\packages\ZXing.Net.Mobile.2.4.1\lib\Xamarin.iOS10\ZXingNetMobile.dll</HintPath>
     </Reference>
+    <Reference Include="PSPDFKit.iOS.Instant">
+      <HintPath>..\..\..\PSPDFKit.iOS.Instant.dll</HintPath>
+    </Reference>
+    <Reference Include="PSPDFKit.iOS.Model">
+      <HintPath>..\..\..\PSPDFKit.iOS.Model.dll</HintPath>
+    </Reference>
+    <Reference Include="PSPDFKit.iOS.UI">
+      <HintPath>..\..\..\PSPDFKit.iOS.UI.dll</HintPath>
+    </Reference>
```

Comment:
I am not sure we was these, since the parent sln should reference by project not by using the dll

Reply:
Fully correct, must've added those accidentally.